modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -310,23 +310,35 @@
 				}
 			);
 
+			var elementFocusable = function(element) {
+				return !element.is(':disabled') && !element.is(':hidden');
+			};
+
 			if (!interacting && Util.inBrowserView(el)) {
 				var form = el.closest('form');
 
-				var focusable = !el.is(':disabled') && !el.is(':hidden');
-
-				if (!form.length || focusable) {
+				if (!form.length || elementFocusable(el)) {
 					el.focus();
 				}
 				else {
 					var portletName = form.data('fm-namespace');
 
-					Liferay.once(
-						portletName + 'formReady',
-						function() {
-							el.focus();
+					var formReadyEventName = portletName + 'formReady';
+
+					var focusOnField = function(event) {
+						if (event) {
```

Comment:
I only added it so the javascript wouldn't break when trying to get `event.formName` in case it wasn't defined, but I suppose it isn't necessary since `event` should always be defined?

Would you like me to remove it and push up the change?